my-policies/policy-zts-cmc-deployment.yaml

```diff
@@ -7,16 +7,80 @@ metadata:
     policy.open-cluster-management.io/categories: CM Configuration Management
```

Comment:
Did you mean to include this directory and file with the PR?  Is this the same policy?  If not, why can't it be included with the other policy as a second example?

Reply:
Removed the my-policies directory and contents.